itou/approvals/notifications.py

```diff
@@ -11,6 +12,7 @@ class NewProlongationToAuthorizedPrescriberNotification(BaseNotification):
 
     def __init__(self, prolongation):
         self.prolongation = prolongation
+        super().__init__(recipients_qs=SiaeMembershipQuerySet)
```

Comment:
Cette classe n'a pas besoin de `recipients_qs` à l'initialisation.

Il y a un unique destinataire de l'email ici https://github.com/betagouv/itou/blob/52ac81deed2ace7230b8b8a4f6a5455888ff872a/itou/approvals/notifications.py#L19 Et vu que le `to` n'est pas un QuerySet mais `self.prolongation.validated_by.email` je n'ai pas pu passer par `recipients_qs`.

En fait je me sert de la classe `BaseNotification` juste pour rassembler les notifs au même endroit…

Peut-être que c'est _overkill_…